src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -6,26 +6,35 @@
 import javax.annotation.Nullable;
 
 import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.util.Assumption;
 import teammates.common.util.Const;
 
 public class FeedbackTextQuestionDetails extends FeedbackQuestionDetails {
 
     @Nullable
     private Integer recommendedLength;
 
+    private Boolean shouldAllowRichText;
+
     public FeedbackTextQuestionDetails() {
         super(FeedbackQuestionType.TEXT);
         recommendedLength = null;
+        shouldAllowRichText = true;
```

Comment:
Minor: I think we should have a test case to check the field is set to true when it get constructed.